src/test.sol

```diff
@@ -133,12 +416,19 @@ contract DSTest {
         } else {
             ok = false;
         }
-
-        if (!ok) {
-            emit log_bytes32("Error: Wrong `bytes' value");
-            emit log_named_bytes32("  Expected", "[cannot show `bytes' value]");
-            emit log_named_bytes32("  Actual", "[cannot show `bytes' value]");
+    }
+    function assertEq0(bytes memory a, bytes memory b) internal {
```

Comment:
why can't this be `assertEq0`?

Reply:
It clashes with `assertEq(string,string)`

Reply:
oh, or it used to hmm

Reply:
oh they have the same abi encoding?

Reply:
you'll get an error like 

```
Error: No unique declaration found after argument-dependent lookup.
   --> aux/demo.sol:212:9:
    |
212 |         assertEq(hex"abcdef01", hex"abcdef02");
    |         ^^^^^^^^
```